distro/systemd/openvpn-client@.service.in

```diff
@@ -3,7 +3,8 @@ Description=OpenVPN tunnel for %I
 After=syslog.target network-online.target
 Wants=network-online.target
 Documentation=man:openvpn(8)
-Documentation=https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage
+Documentation=https://openvpn.net/community-resources/reference-manual-for-openvpn-@OPENVPN_VERSION_MAJOR@-@OPENVPN_VERSION_MINOR@/
+Documentation=https://build.openvpn.net/man/openvpn-@OPENVPN_VERSION_MAJOR@.@OPENVPN_VERSION_MINOR@/openvpn.8.html
```

Comment:
Both links point to the same document (the HTML version of the man page). So we can drop the second one.

Reply:
Thanks, I updated the change.